third_party/silabs/silabs_executable.gni

```diff
@@ -29,9 +28,13 @@ template("silabs_executable") {
   # even if the build and flashing steps take place on different machines
   # or in different containers.
 
+  if (use_rps_extension) {
+    flashing_image_name = output_base_name + ".rps"
```

Comment:
this breaks the flasher for 917 Socs if not built with our bash script

e.g: 
- gn /ninja command line
- build_examples.py

Reply:
fixed